src/modules/ZoomIt/ZoomIt/Zoomit.cpp

```diff
@@ -7416,19 +7416,19 @@ int APIENTRY wWinMain(_In_ HINSTANCE hInstance, _In_opt_ HINSTANCE hPrevInstance
     if( !ShowEula( APPNAME, NULL, NULL )) return 1;
 
 #ifdef __ZOOMIT_POWERTOYS__
+    if (powertoys_gpo::getConfiguredZoomItEnabledValue() == powertoys_gpo::gpo_rule_configured_disabled)
```

Comment:
how to enforce enabled state? 🤔

Reply:
You can't on the executable itself. 😅 But will run from PowerToys if you start PowerToys with the enabled state.